.dockerignore

```diff
@@ -1,6 +1,6 @@
 target/
 build/
 .github/
-Dockerfile.build
+build.Dockerfile
```

Comment:
Is this still needed with `Dockerfile.build` is getting removed?

Reply:
Fixed the gitignore rules, and readded the file :)